code/modules/tents/deployed_tents.dm

```diff
@@ -71,28 +73,62 @@
 
 /obj/structure/tent/proc/mob_exited_tent(mob/subject)
 	UnregisterSignal(subject, list(COMSIG_MOVABLE_TURF_ENTERED, COMSIG_GHOST_MOVED, COMSIG_HUMAN_COLD_PROTECTION_APPLY_MODIFIERS))
+	subject.UnregisterSignal(src, COMSIG_PARENT_QDELETING)
 	var/atom/movable/screen/plane_master/roof/roof_plane = subject.hud_used.plane_masters["[ROOF_PLANE]"]
 	roof_plane?.invisibility = 0
 
 /obj/structure/tent/attack_alien(mob/living/carbon/xenomorph/M)
 	if(unslashable)
 		return
-	health -= 20
+
+	M.animation_attack_on(src)
```

Comment:
When we talked about it and i tested the animation / sound was already played, it's just confusing currently since you can't click through the blockers

Reply:
It made the sounds for a "null" attack, same sound as missing your attack and hitting the floor instead.
This will make it clearer